src/frontend/screens/Settings/ProjectSettings/YourTeam/SelectDevice.tsx

```diff
@@ -0,0 +1,83 @@
+import {defineMessages, useIntl} from 'react-intl';
+import {NativeNavigationComponent} from '../../../../sharedTypes';
+import {ScrollView, StyleSheet, View} from 'react-native';
+import WifiIcon from '../../../../images/WifiIcon.svg';
+import {Text} from '../../../../sharedComponents/Text';
+import {DeviceCard} from '../../../../sharedComponents/DeviceCard';
+import {useLocalDiscoveryState} from '../../../../hooks/useLocalDiscoveryState';
+import {useLocalPeers} from '../../../../hooks/useLocalPeers';
+
+const m = defineMessages({
+  title: {
+    id: 'screen.Settings.ProjectSettings.YourTeam.SelectDevice.title',
+    defaultMessage: 'Select Device to Invite',
+  },
+  notSeeingDevice: {
+    id: 'screen.Settings.ProjectSettings.YourTeam.SelectDevice.notSeeingDevice',
+    defaultMessage: 'Not seeing a Device?',
+  },
+  sameWifi: {
+    id: 'screen.Settings.ProjectSettings.YourTeam.SelectDevice.sameWifi',
+    defaultMessage: 'Make sure both devices are on the same wifi network ',
+  },
+  sameVersion: {
+    id: 'screen.Settings.ProjectSettings.YourTeam.SelectDevice.sameVersion',
+    defaultMessage: 'Make sure both devices are on the same version of Mapeo',
+  },
+});
+
+export const SelectDevice: NativeNavigationComponent<'SelectDevice'> = ({
+  navigation,
+}) => {
+  const ssid = useLocalDiscoveryState(state => state.ssid);
+  const {formatMessage: t} = useIntl();
+
+  const devices = useLocalPeers();
```

Comment:
Note, this will include devices that are already members of the project until #167 is fixed